### DISCUSSION TITLE
Creating rules with parts of class names

### DISCUSSION THREAD

**Discussion by DimitrisKalaboukas** [uBlockOrigin/uAssets#25860] [Q&A] [answered]
How can I transform rules like:

dropbox.com##._suggestFrame_7mov9_72

in order to mention only stable “_suggestFrame_” string, because the following part changes very frequently?

**Comment by stephenhawk8054** [comment, marked as answer]
```adb
dropbox.com##[class*="_suggestFrame_"]
```

?

**Comment by garry-ut99** [comment]
> DimitrisKalaboukas : How can I (...)

Duplicate of:

- https://github.com/uBlockOrigin/uBlock-issues/issues/3352#issuecomment-2302002052

Be nice and don't spam with basic questions which are already answered hundreds of times, just read documentation, manuals, wiki, issue tracker and so on, devs's time is limited.

**Comment by gcb** [comment]
Don't know why question is being downvoted. This is extremely handy.

e.g.
```
*##[class*="accordion"]:style(display: initial! important; height: initial !important; visibility: initial !important;)
*##[class*="collapse"]:style(display: initial! important; height: initial !important; visibility: initial !important;)
```

**Reply by freezer2022** [reply]
> gcb:  Don't know why question is being downvoted.

The explanation is already provided in the hidden comment, just above your comment.

**Reply by gcb** [reply]
oh thanks for pointing it out. Missed it entirely
